src/simModel/TalkToOperator.java

```diff
@@ -0,0 +1,41 @@
+package simModel;
+
+import simulationModelling.ConditionalActivity;
+
+class TalkToOperator extends ConditionalActivity {
+    SMTravel model;
+    //Customer Type is definied in Call.java, double check
+    private Call uCustomerType;
+
+    public TalkToOperator(SMTravel model) { this.model = model; }
+
+    protected static boolean precondition(SMTravel simModel){
+        boolean returnValue = false;
+        //this may need to be modified based on other classes
+        if(simModel.rgOperator.type == simModel.n && simModel.n != 0){
```

Comment:
this need to use the SMTravel instead of simModel. what simModel.n is and how this will compare with the type and type accept a 0 value